main.tex

```diff
@@ -16,8 +16,43 @@
 }
 
 \begin{document}
+\subsection{Beweggründe für Internet.org}
+Facebook, ein soziales Netzwerk, wurde 2004 gegründet und ist seit dem, genauso wie das dahinterstehende Unternehmen Facebook, Inc., stetig gewachsen. 
+Doch da die Anzahl von Anmeldungen neuer Internet-Kabelanschlüsse in der westlichen Welt seit 2008 gesunken sind,
+(Vgl. \cite{ICTslowingDown}) hat das Wachstum von Facebook dementsprechent auch nachgelassen.\\
+Dies passiert obwohl ``at the beginning of 2016, only an estimated 3.2 billion people — 44 percent of the world’s population are online and connected to the digital economy." (\cite{connectWorld})\\
+Der Grund dafür: der Großteil der restlichen 56\% der Menschen lebt in Entwicklungsländern. Dort hat man meist entweder nicht die Möglichkeit das Internet zu nutzen (wegen zu hoher Kosten/keinen Internetzugang) oder man sieht den Nutzen des Internets nicht, da dieser nie vermittelt worden ist.
 
-\maketitle
+\medskip
 
+Für Facebook, eine Firma welche durch Werbung und Personenbezogenen Daten Geld verdient, ist es von langfristiger Interesse, dass sich die Anzahl von Menschen, welche das Internet benutzen, erhöht.
+Aus den vorherigen Pragraph lässt sich ableiten, dass sich dafür das Internet in den Entwicklungsländern ausbreiten, und die Interesse der Menschen für die Anwendung geweckt werden muss.
+Facebook ist nicht die einzige IT-Firma welche Interesse daran gezeigt hat.
+Eine weitere ist die Dachgesellschaft von Google, sie hat 2011 mit ihren Tests für das Projekt Loon, welches das Ziel hat das Internet Netzwerk auf abgelegende Gegenden auszubreiten, angefangen.(Vgl. \cite{projectLoon})
```

Comment:
```suggestion
Eine weitere ist die Dachgesellschaft von Google, Alphabet Inc., welche mit dem Projekt Loon das Ziel verfolgt Internet in abgelegenden Gegenden auszubreiten (Vgl. \cite{projectLoon}).
```

Reply:
Anstatt Projekt Loon, kann vlt. lieber Bezug auf [googles RailWire](https://gadgets.ndtv.com/internet/features/google-railwire-railtel-india-free-internet-train-stations-1731879) (auch in Indien) genommen werden.

Reply:
Das wird nämlich heute auch benutzt